node/test/unit/tessel.js

```diff
@@ -180,6 +180,19 @@ exports['Tessel.LED'] = {
     test.done();
   },
 
+  isOn: function(test) {
+    test.expect(3);
+
+    test.equal(this.tessel.led[0].isOn, false);
+
+    this.tessel.led[0].on();
+
+    test.equal(this.tessel.led[0].isOn, true);
+
+    test.throws(this.tessel.led[0].isOn = false);
```

Comment:
Ah thanks for pointing that out. I wasn't sure if I had that setup correctly but the tests passed, so I figured it was correct. I'll commit a quick fix for that.